packages/openapi-v3-types/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@loopback/openapi-v3-types",
+  "version": "4.0.0-alpha.1",
```

Comment:
Can we use `1.0.0-alpha.1` since this module is not really tied to LB4 as far as version numbers are concerned?

Reply:
sure that sounds reasonable 👍